assets/css/paper.css

```diff
@@ -41,13 +41,19 @@ nav {
 	background-color: #f19fad;
 
 }
-
+.head{
+	font-size: 48px;
+	background-color: #F100AE;
```

Comment:
Drop the background color for the header.

Reply:
I thought the background color would make the header and footer look different from the main content, but OK, I will do it.